paasta_tools/cli/cmds/status.py

```diff
@@ -158,7 +158,8 @@ def report_status_for_cluster(service, cluster, deploy_pipeline, actual_deployme
 
     if len(deployed_instances) > 0:
         status = execute_paasta_serviceinit_on_remote_master('status', cluster, service, ','.join(deployed_instances),
-                                                             system_paasta_config, stream=True, verbose=verbose)
+                                                             system_paasta_config, stream=True, verbose=verbose,
+                                                             ignore_ssh_output=True)
```

Comment:
What is the point of this new `ignore_ssh_output`? I don't see it getting used here.